Cargo.toml

```diff
@@ -20,6 +20,7 @@ serde_json = { version = "1" }
 cfg-if = "1"
 hex = "0.4"
 hex-literal = "0.3.4"
+thiserror = "1.0"
```

Comment:
I forgot to add some context to this change suggestion. We fix the Major and the Minor and not the Patch to make sure that we are using the last patched version of the crate.